nss-day-cohort-47/tabloid-mvc-reese-s-puffs
# Mc delete post

# Description
A User can now Delete and Edit a post they made.
I also re-added  the method that allows users to see their posts

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change


# Testing Instructions
Go to Posts in the nav, then try deleting a post in the list. Once you have deleted a post try editing a post.

Lastly click on MY POSTS in the nav and just make sure the page populates with the users posts


# Checklist:
